app/controllers/system_admin/providers_controller.rb

```diff
@@ -25,6 +25,10 @@ def create
     def show
       @provider = authorize Provider.find(params[:id])
       @users = @provider.users.order(:last_name)
+      @users_not_registered = Dttp::User.not_registered_with_provider(
```

Comment:
Once there is more than one instance variable being used I think we should generally consider using a view object. In this method we're not testing the ordering or that we're getting the correct users for the view. We could do `User.all` and `Dttp::User.all` here and the feature spec would still pass. Using a view object we could easily unit test those things.